src/App.tsx

```diff
@@ -1,61 +1,42 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
```

Comment:
The direct import of `usersFromServer` here should be avoided. Instead, prepare the data in the `App` component and pass it down to the `TodoForm` component as props.